strings/tips/marine.txt

```diff
@@ -1,80 +1,79 @@
 The Revolver and Service Pistol can be decent primary weapons if used properly, though with the short magazine capacity, expect to run dry on ammo quick!
 There is no "best" loadout. Experiment and find what works for you.
 You have a variety of weaponry inside the Automated Weapon Closets, do not be afraid to experiment which one is best suited for your playstyle.
-There are "specialist" weaponry that can be ordered by Requisitions. However, they take quite an fairly high amount to order them, what's more, if the user dies irresponsibly while using these weapons themselves, it will be a massive waste of points.
+There is "specialist" weaponry that can be ordered by Requisitions. However, they take a fairly high amount of points to order them. If the user dies irresponsibly while using these weapons, it will be a massive waste of points.
 Anything can be a weapon if you put your mind to it.
 You can check your wounds to see if they have been bandaged or salved by clicking on yourself while in help intent.
 You can check what part of your body is bleeding by shift-clicking on your person.
-Removing weeds as a marine means that xenomorphs cannot take advantages of the battlefield; your standard boot knife is great for doing this.
-Green weeds and purple sticky resin slow you down. Get rid of them!
+Removing weeds as a marine means that xenomorphs cannot take advantage of the battlefield; your standard boot knife is great for doing this.
+Blue weeds and purple sticky resin slow you down. Get rid of them!
 During Self Destruct, green control rods are armed control rods. Each rods take 3 minutes to rise, forcing marines to hold Self Destruct for a total of 20 minutes. Collect all six!
 A fit marine can carry many weapons: in their armor, belt, even hanging from their back.
-Requisitions have many supplies to enhance the combat power of marine units, such as attachments, ammunition, and other toys; ask their crew what's on store and you may be pleasantly surprised. DISCLAIMER: Extra gear's availability is dependent on supply.
-Cargo points are valuable: Some crates, like SL armour, standard weaponry or attachments, may be a waste of points which could be better spent on other stuff.
-As a human, you can climb over waist-high obstacles like sandbags, window frames or tables by "SpecialClicking" (Default: CTRL) on them or drag-clicking yourself on them.
+Requisitions has many supplies to enhance the combat power of marine units, such as attachments, ammunition, and other toys; ask their crew what's in store and you may be pleasantly surprised. DISCLAIMER: Extra gears' availability is dependent on supply.
+Cargo points are valuable: some crates, like SL armour, standard weaponry or attachments, may be a waste of points which could be better spent on other stuff.
+As a marine, you can climb over waist-high obstacles like sandbags, window frames or tables by "SpecialClicking" (Default: CTRL) on them or drag-clicking yourself onto them.
```

Comment:
should probs mention jumping.

Reply:
i went ahead and made it a new tip, just because i think combining the two would be too long, and i wanted to flesh out more than just "you can also jump". i can also just delete the "specialclicking" tip since i think jumping is the superior way to move across objects

```You can jump over waist-high obstacles like platforms, tables, and windowframes, for a small stamina cost, or, if a xeno, for a two-second cooldown. Set your key to jump under the Game Preferences menu!```

let me know if you want the wording edited at all. i also forgot, but it has to be binded, there's no default key, correct?

Reply:
but default it should be space or something.